modules/floating-buttons/widgets/contact-buttons.php

```diff
@@ -3,6 +3,7 @@
 namespace Elementor\Modules\FloatingButtons\Widgets;
 
 use Elementor\Modules\FloatingButtons\Base\Widget_Contact_Button_Base;
+use Elementor\Plugin;
```

Comment:
Do we need this import?

Reply:
Yes!

It fixed an error in the CI. See: https://github.com/elementor/elementor/pull/28128/commits/daa422531d436cf6761e2e13ecbd78ef11536ad5 commit.

Reply:
Don't remember which error, but it took me a while to find the issue.